packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -374,6 +374,8 @@ void Jacobi(Scalar omega,
       int mm_optimization_core_count=0;
       auto slist = params->sublist("matrixmatrix: kernel params",false);
       mm_optimization_core_count = slist.get("MM_TAFC_OptimizationCoreCount",::Tpetra::Details::Behavior::TAFC_OptimizationCoreCount ());
+      int foo = params->get("MM_TAFC_OptimizationCoreCount",mm_optimization_core_count);
+      if(foo<mm_optimization_core_count) mm_optimization_core_count=foo;
```

Comment:
So `MM_TAFC_OptimizationCoreCount` could be either in the parent or child ParameterList?

Reply:
@DrBooom had mentioned something about this being on purpose.

Reply:
@DrBooom had mentioned this being on purpose.   Could we call this something other than `foo`, please?